*: bump github.com/linode/linodego to 1.1.0 and tidy go.sum

paulfantom opened the pull request:
This would fix dependabot upgrades. More in https://github.com/timescale/promscale/network/updates/216242223

cevian reviewed (approved):
Looks good. One minor tweak: for commit messages we use this convention: https://chris.beams.io/posts/git-commit/ 

So please change commit message to "Upgrade dependencies to fix dependabot" or something similar

paulfantom commented:
Fixed commit message to be more descriptive and rebased as there was merge conflict.

paulfantom commented:
Fixed commit message to be more descriptive and rebased as there was merge conflict.

BTW. There is something weird going on with `github.com/shurcooL/vfsgen` dependency as it is required for one of generators, but it is discarded by golang build chain when doing `go mod tidy`. :thinking:

paulfantom commented:
BTW, funny how https://chris.beams.io/posts/git-commit/ contradicts official golang style guide https://golang.org/doc/contribute#commit_messages in some places :smile:
